components/Transactions/CoinPicker.tsx

```diff
@@ -6,6 +6,9 @@ import {
   ITransactionsContext,
 } from "../../providers/TransactionsProvider";
```

Comment:
Can we move this provider down, to put all the providers together last?

Reply:
ok i'm gonna do that